.pre-commit-config.yaml

```diff
@@ -1,17 +1,21 @@
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.2.0
+    rev: v4.4.0
```

Comment:
Ran ` pre-commit autoupdate`  to update these. If for some reason you want a specific version (maybe clang-format needs to be version 14?), we can revert of course.

Reply:
Not really, just copy pasted what was on the official documents